src/main/menu/application-menu.injectable.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable } from "@ogre-tools/injectable";
+import { autorun } from "mobx";
+import { buildMenu } from "./menu";
+import applicationMenuItemsInjectable from "./application-menu-items.injectable";
+import { getStartableStoppable } from "../../common/utils/get-startable-stoppable";
+
+const applicationMenuInjectable = getInjectable({
+  id: "application-menu",
+
+  instantiate: (di) => {
+    const applicationMenuItems = di.inject(applicationMenuItemsInjectable);
+
+    return getStartableStoppable("build-of-application-menu", () =>
```

Comment:
When would we want to stop building this?

Reply:
Do we want to rely on electron not having memory leaks and actually removing all traces as it should? In perfect world, sure. 

Our code should always clean-up these type of things before we quit the application. 

This is exact same behaviour than before. 

I agree with terminology.